Testscripts/Linux/SetupHbKernel.sh

```diff
@@ -20,18 +20,16 @@ GetDistro
 # Get generation: 1/2
 GetGuestGeneration
 
-# Hibernation is supported in RHEL-8 since kernel-4.18.0-202. Not supported in RHEL-7.
-if [[ $DISTRO =~ "redhat" ]];then
-	MIN_KERNEL="4.18.0-202"
-	CheckVMFeatureSupportStatus $MIN_KERNEL
-	if [[ $? == 1 ]];then
-		UpdateSummary "Hibernation is supported since kernel-4.18.0-202. Current version: $(uname -r). Skip the test."
-		SetTestStateSkipped
-		exit 0
+function Main() {
+	basedir=$(pwd)
```

Comment:
I can see different variable naming styles throughout the script. (variableName and variable_name).

Reply:
Changed in 5fbfc4db784b77da63f3e8a5c402175e973af791